requirements/constraints.txt

```diff
@@ -25,6 +25,9 @@ django-storages<1.9                 # BSD
 # django-waffle version 0.19.0 dropped support to Django 2.0 and 2.1, see https://github.com/django-waffle/django-waffle/blob/master/CHANGES
 django-waffle<0.19.0
 
+# django22 tests are failing. Constraint taken from https://github.com/edx/edx-drf-extensions version 4.0.1
```

Comment:
if this is handled by edx-drf-extensions, why should we handle this again, if we are not added the requirement directly ?

Reply:
https://github.com/edx/edx-drf-extensions requirements are taken from https://github.com/edx/edx-drf-extensions/blob/master/requirements/base.in. 

In that file there are not contraints for drf-jwt, so pip-compile tools is not aware of that. For that reason I had to add that to the constraints.

Nevertheless, I think that this should be discussed in https://github.com/edx/edx-analytics-data-api/pull/326

Reply:
If I not add that constraint, then the tests fails because the last version of drf-jwt requires some changes to the code of edx-drf-extensions.

Reply:
got it